Proteomics/Modifications/ModificationGeneral.cs

```diff
@@ -48,28 +62,33 @@ public ModificationGeneral(string _Id, string _Accession, string _ModificationTy
             this.FileOrigin = _FileOrigin;
         }
 
-        public static ModLocationOnPeptideOrProtein? Localization(string myLocalization)
+        public static string ModLocationOnPeptideOrProtein(string modLocation)
```

Comment:
1. the original switch statement is more efficient; just change the default return value from null to "Unassigned."
2. change to return an enumerated type instead of the string

Reply:
switched to switch but using an enumerated type ends up being a pain in the neck. i did that for a while then changed everything back. easier to explain in person.